sdk/python/feast/feature_view.py

```diff
@@ -213,36 +213,16 @@ def ensure_valid(self):
     def proto_class(self) -> Type[FeatureViewProto]:
         return FeatureViewProto
 
-    def with_name(self, name: str):
```

Comment:
Are we sure this method isn't being used anywhere? I'd check with @mavysavydav in case they rely on it internally.

Reply:
hm I think this method is inherited from `BaseFeatureView`, so should be fine? same with below method